cmd/queue/main.go

```diff
@@ -247,9 +250,15 @@ func probeQueueHealthPath(port int, timeoutSeconds int) error {
 	// Using PollImmediateUntil instead of PollImmediate because if timeout is reached while waiting for first
 	// invocation of conditionFunc, it exits immediately without trying for a second time.
 	timeoutErr := wait.PollImmediateUntil(aggressivePollInterval, func() (bool, error) {
-		var res *http.Response
-		if res, lastErr = httpClient.Get(url); res == nil {
```

Comment:
Is `lastErr` no longer used? Can we remove the declaration above and the check below?

Reply:
Good caught. It returns nil error in `PollImmediateUntil` for retrying and records `lastErr` if it really gets an error. I'll keep it.